internal/worker/rundynamic.go

```diff
@@ -108,6 +110,28 @@ func addSSHKeysToSandbox(ctx context.Context, sb sandbox.Sandbox) error {
 	return sb.CopyIntoSandbox(ctx, tempdir+"/.", "/root/.ssh")
 }
 
+// generateAWSAccessID generates an AWS access key id based off of some known patterns and random values.
+func generateAWSAccessKeyId() (string, error) {
+	const charSet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ234567"
+	var accessKeyId = "AKIAI"
+	for i := 0; i < 14; i++ {
+		randInt, err := rand.Int(rand.Reader, big.NewInt(int64(len(charSet))))
```

Comment:
On further thought, we can probably just use the [`rand.Intn()`](https://pkg.go.dev/math/rand#Rand.Intn) function from the `math/rand` package, instead of the functions from `crypto/rand`.  

Since we're not generating a cryptographic key (it just has to look like one), we don't need the stronger guarantees of the `crypto/rand` package. 

Additionally, the `math/rand` package functions don't require error handling and the API is simpler. This mean we can probably combine the generation of both env vars into a single function. 

We can create a  `rand.Rand` instance to use here from a [`Source`](https://pkg.go.dev/math/rand#Source) which is seeded using the global [`Int63()`](https://pkg.go.dev/math/rand#Source) function.